### PR TITLE
correct bubble position

### PR DESCRIPTION
**Pull request by Bayramito**
This PR centers the bubble correctly
Ekran Resmi 2024-02-29 09 00 12

### CODE REVIEW

**Comment by alantoa** [comment]
Thanks for your PR. I'll test a bit and then merge it.

File: src/slide.tsx [hunk 1]
```diff
@@ -910,7 +910,7 @@ export const Slider: FC<AwesomeSliderProps> = memo(function Slider({
           style={[
             styles.bubble,
             {
-              left: -bubbleMaxWidth / 2,
+              left: -bubbleMaxWidth / 2 - thumbWidth / 2,
```
Comment (ArindamRayMukherjee) [inline src/slide.tsx:913]: Makes sense :) 

@alantoa Chance of a quick merge and release?

**Comment by alantoa** [comment]
thanks you guys! just released a new version! https://github.com/alantoa/react-native-awesome-slider/releases/tag/v2.5.2

**Comment by alantoa** [comment]
I just noticed that this PR is not correct, so I reverted it.
If you want to correct the position of the bubble, you need to preset the `thumbWidth` prop. For example, in your example image, you should set `thumbWidth={1}` in the Slider component. The bubble will automatically calculate its position.

**Pull request merged**